lib/MusicBrainz/Server/Data/WikiDoc.pm

```diff
@@ -147,7 +147,7 @@ sub _load_page
         return undef;
     }
 
-    if ($content =~ /<span class="redirectText"><a href="(?:https?:)?\/\/.*?\/(.*?)"/) {
+    if ($content =~ m{<ul class="redirectText"><li><a href="(?:https?:)?(?://[^/]+)?/(.*?)"}) {
```

Comment:
There were two changes: 1. the HTML elements, 2. the URL is currently a relative one (but this should work for relative, schema-relative and absolute URLs). I also used `m{}` instead of `//` so that slashes don’t need to be escaped.